### PR TITLE
Swagger 205 remove van and iban endpoints

### PR DESCRIPTION
**Pull request by maria-thom**

### CODE REVIEW

File: src/reference.yaml [hunk 1]
```diff
@@ -14902,164 +14712,7 @@ paths:
...
-              type: string
-              description: A unique reference for the request.
-        default:
-          description: Unexpected error.
-          headers:
-            X-Request-Id:
-              type: string
-              description: A unique reference for the request.
-  /withdrawal_accounts/find:
+    /withdrawal_accounts/find:
```
Comment (jonathancouchman) [inline src/reference.yaml:14715]: This resource is now mis-aligned. delete the additional spaces

**Review by jonathancouchman — CHANGES_REQUESTED** [review]

File: src/reference.yaml (hunk 1 above)
Comment (maria-thom) [reply src/reference.yaml]: Thanks - that's sorted it 👍

**Review by jonathancouchman — CHANGES_REQUESTED** [review]
You need to also remove the referenced objects that are no longer used

/definitions/GetIBANsError
/definitions/IBAN
/definitions/GetVANsError
/definitions/VAN

**Comment by maria-thom** [comment]
> You need to also remove the referenced objects that are no longer used
> 
> /definitions/GetIBANsError
> /definitions/IBAN
> /definitions/GetVANsError
> /definitions/VAN

**Review by jonathancouchman — APPROVED** [review]

**Pull request merged**